config/config_helpers.go

```diff
@@ -115,6 +115,7 @@ func CreateTerragruntEvalContext(
 		"get_terraform_commands_that_need_input":       wrapStaticValueToStringSliceAsFuncImpl(TERRAFORM_COMMANDS_NEED_INPUT),
 		"get_terraform_commands_that_need_parallelism": wrapStaticValueToStringSliceAsFuncImpl(TERRAFORM_COMMANDS_NEED_PARALLELISM),
 		"sops_decrypt_file":                            wrapStringSliceToStringAsFuncImpl(sopsDecryptFile, extensions.Include, terragruntOptions),
+		"get_terragrunt_source":                        wrapVoidToStringAsFuncImpl(getTerragruntSource, extensions.Include, terragruntOptions),
```

Comment:
Perhaps we should rename this to `get_terragrunt_source_cli_flag` to make clear that this only returns values passed through with `--terragrunt-source`?